src/QueryEditor.tsx

```diff
@@ -2,9 +2,10 @@
  * QueryEditor is a React component that implements the UI for building a notebook query
  * when editing a Grafana panel.
  */
+import _ from 'lodash';
```

Comment:
Can you consolidate the lodash usages? I think because we were only using a single method I imported it directly.
I'm talking about this line: `import pickBy from 'lodash/pickBy';`
You can remove that import and update the `pickBy` call to `_.pickBy`.